client/js/lib/prosemirror/plugins.js

```diff
@@ -336,39 +336,57 @@ const rangeTracker = (rangeTrackerKey, schema, rangeChangeCallback = () => {}) =
  */
 const rangeCreator = (pluginKey, rangeEditingKey) => {
   let tippy = null
+
+  const handleGlobalMouseup = (view, e) => {
+    const { state } = view
+    const { selection } = state
+    const { empty, from, to, $anchor, $head } = selection
+
+    if (empty) {
+      tippy?.destroy()
+      tippy = null
+      return
+    }
+
+    const lastClick = view.lastClick || view.input.lastClick
+    const hasClickLocationChanged = (e.clientX !== lastClick.x || e.clientY !== lastClick.y)
+
+    if (hasClickLocationChanged) {
+      const existingRanges = rangeEditingKey.getState(state)
+      let positionsCovered = []
+      Object.values(existingRanges).forEach(({ from, to }) => positionsCovered.push(...range(from, to)))
+      const selectedPositions = new Set(range(from, to))
+      positionsCovered = new Set(positionsCovered)
+      const isFullyCovered = isSuperset(positionsCovered, selectedPositions)
```

Comment:
For me this bock is not so good to read. But If You don't like my suggestion, feel free to drop it.
```suggestion
      const existingRanges = rangeEditingKey.getState(state)
      const selectedPositions = new Set(range(from, to))
      let positionsCovered = []
      
      Object.values(existingRanges).forEach(({ from, to }) => positionsCovered.push(...range(from, to)))
      const isFullyCovered = isSuperset(new Set(positionsCovered), selectedPositions)
```